x-pack/test/detection_engine_api_integration/basic/tests/delete_rules_bulk.ts

```diff
@@ -59,8 +60,9 @@ export default ({ getService }: FtrProviderContext): void => {
         // delete that rule by its rule_id
         const { body } = await supertest
           .delete(DETECTION_ENGINE_RULES_BULK_DELETE)
-          .send([{ rule_id: bodyWithCreatedRule.rule_id }])
           .set('kbn-xsrf', 'true')
+          .set('elastic-api-version', '2023-10-31')
+          .send([{ rule_id: bodyWithCreatedRule.rule_id }])
```

Comment:
Nice catch! So is setting `kbn-xsrf` not necessary here? Why no issue before if it was previously `send`'ing before setting it, or does supertest batch till the `expect`?